sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java

```diff
@@ -6,61 +6,26 @@
  */
 package com.farao_community.farao.sensitivity_computation;
 
-import com.powsybl.commons.config.ComponentDefaultConfig;
-import com.powsybl.computation.ComputationManager;
-import com.powsybl.computation.DefaultComputationManagerConfig;
 import com.powsybl.iidm.network.Network;
 import com.powsybl.sensitivity.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
-
 /**
  * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
  */
 final class SystematicSensitivityService {
```

Comment:
Maybe we can rename into SystematicSensitivityAdapter :) Even merge it into SystematicSensitivityInterface as it is really small now

Reply:
done